feature/search/src/main/java/com/terning/feature/search/search/component/ImageSlider.kt

```diff
@@ -51,30 +54,35 @@ fun ImageSlider(
             .background(Grey200),
         horizontalAlignment = Alignment.CenterHorizontally
     ) {
-        Box(
-            modifier = modifier,
-            contentAlignment = Alignment.BottomCenter
-        ) {
-            HorizontalPager(
-                state = pagerState,
+        if (searchBanners.isNotEmpty()) {
+            Box(
                 modifier = modifier,
-                beyondViewportPageCount = 1
-            ) { currentPage ->
-                val pageIndex = currentPage % images.size
-                Image(
-                    painter = painterResource(id = images[pageIndex].imageRes),
-                    contentDescription = null,
-                    modifier = modifier
-                        .fillMaxWidth()
-                        .height(112.dp)
-                        .noRippleClickable { onAdvertisementClick(pageIndex) },
-                    contentScale = ContentScale.Crop,
+                contentAlignment = Alignment.BottomCenter
+            ) {
+                HorizontalPager(
+                    state = pagerState,
+                    modifier = modifier,
+                    beyondViewportPageCount = 1
+                ) { currentPage ->
+                    val pageIndex = currentPage % searchBanners.size
+                    AsyncImage(
+                        model = ImageRequest.Builder(LocalContext.current)
+                            .data(searchBanners[pageIndex].imageUrl)
+                            .crossfade(true)
+                            .build(),
+                        contentDescription = null,
+                        modifier = modifier
+                            .fillMaxWidth()
+                            .height(112.dp)
+                            .noRippleClickable { onAdvertisementClick(pageIndex) },
+                        contentScale = ContentScale.Crop,
```

Comment:
여기 두 부분 모두 Modifier로 바꿔줘야 할 것 같아요,,!!
```suggestion
                HorizontalPager(
                    state = pagerState,
                    modifier = Modifier,
                    beyondViewportPageCount = 1
                ) { currentPage ->
                    val pageIndex = currentPage % searchBanners.size
                    AsyncImage(
                        model = ImageRequest.Builder(LocalContext.current)
                            .data(searchBanners[pageIndex].imageUrl)
                            .crossfade(true)
                            .build(),
                        contentDescription = null,
                        modifier = Modifier
                            .fillMaxWidth()
                            .height(112.dp)
                            .noRippleClickable { onAdvertisementClick(pageIndex) },
                        contentScale = ContentScale.Crop,
```